src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -131,7 +131,29 @@ public function is_vcs_checkout( $context ) {
 	 *
 	 * @param string $type    The type of update being checked: 'core', 'theme',
 	 *                        'plugin', 'translation'.
-	 * @param object $item    The update offer.
+	 * @param object $item {
+	 *     The update offer.
+	 *
+	 *     @type string   $current            The version number of the update offer.
+	 *     @type bool     $disable_autoupdate Whether automatic updates for this particular update offer have been
+	 *                                        remotely disabled via the update API response.
+	 *     @type bool     $dismissed          Whether this update offer has previously been dismissed.
+	 *     @type string   $download           URL to the ZIP file for the update.
+	 *     @type string   $locale             Locale of the update offer, eg. `en_US`.
+	 *     @type string   $mysql_version      Minimum required MySQL version for the update.
+	 *     @type object   $packages {
+	 *         URLs for various ZIP packages for the update.
+	 *
+	 *         @type false|string $full        URL to the full ZIP package.
+	 *         @type false|string $no_content  URL to the no content ZIP package.
+	 *         @type false|string $new_bundled URL to the new bundled ZIP package.
+	 *         @type false|string $partial     URL to the partial ZIP package.
+	 *         @type false|string $rollback    URL to the rollback ZIP package.
+	 *     }
+	 *     @type string   $php_version        Minimum required PHP version for the update.
+	 *     @type string   $plugin             File name of the plugin in the update offer, eg. `akismet/akismet.php`.
+	 *     @type string   $theme              Directory name of the theme in the update offer, eg. `twentytwenty`.
```

Comment:
Could instances of eg. be changed to something without the abbreviation? It seems that "Example: akismet/akismet.php" is more common in Core.

I actually think we should try and remove all other occurrences of `eg.` within docblocks. I'll open a ticket for that.